includes/lib/transifex-live-integration-wp-services.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+/*
+ * Language rewrites
+ * @package TransifexLiveIntegration
+ */
+
+/**
+ * Provides WP services to other classes.
+ */
+class Transifex_Live_Integration_WP_Services {
+
+    /*
+	 * Wraps WP site_url().
+	 * @return string Site URL link with optional path appended.
```

Comment:
Where is the `optional path appended` part in the code?

Reply:
I just copied the docstring from the site_url() source code and this created the misunderstanding: https://core.trac.wordpress.org/browser/tags/5.1.1/src/wp-includes/link-template.php#L3141. For this I'll remove the @return line altogether.